### DISCUSSION TITLE
ESP Home mit Home Assistant

### DISCUSSION THREAD

**Discussion by Dobs0001** [frickelzeugs/FingerprintDoorbell#44] [Q&A]
Hallo,  
Wie bekomme ich den Fingerprint in den Home Assistant integriert? 
Bekomme es mit dem ESP Home nicht hin.
Stelle ich mich zu blöd an?  Bin neu in der Materie ESP usw.

vielleicht kann mir jemand helfen. 

Irgendwie fehlt die API in dem Code.

**Comment by mkrasselt1** [comment]
Das ist auch nicht für ESP-Home gedacht sondern für MQTT - du musst also selbst ein MQTT Gerät anlegen mit den Topics und auf gesendete Nachrichten Reagieren / bzw - selber welche senden

**Reply by janboesebeck** [reply]
Kannst du mir eventuell erklären wie ich die einzelnen Punkte im MqTT Mosquitto bei Homeassistant anlege ?
